diff --git src/file_io.cpp src/file_io.cpp
index 2857b0e..13d2528 100644
--- src/file_io.cpp
+++ src/file_io.cpp
@@ -127,7 +127,7 @@ void PropColumn::sqlite3_result(const RDKit::ROMol & mol, sqlite3_context * ctx)
         assert(!"Unexpected value for Type enum");
     }
   }
-  catch (const boost::bad_any_cast & e) {
+  catch (const std::bad_any_cast & e) {
     chemicalite_log(SQLITE_MISMATCH, "could not convert the mol property to the requested type");
     sqlite3_result_error_code(ctx, SQLITE_MISMATCH);
   }
diff --git src/mol_props.cpp src/mol_props.cpp
index c7680a2..47ded83 100644
--- src/mol_props.cpp
+++ src/mol_props.cpp
@@ -306,7 +306,7 @@ static void mol_get_prop(sqlite3_context* ctx, int /*argc*/, sqlite3_value** arg
     try {
       F(ctx, *mol, key);
     }
-    catch (const boost::bad_any_cast & e) {
+    catch (const std::bad_any_cast & e) {
       sqlite3_result_error_code(ctx, SQLITE_MISMATCH);
     }
   }
